### ISSUE TITLE
tensorflow-tensorboard doesn't build with Python 3

### ISSUE THREAD

**Issue by IQubic** [NixOS/nixpkgs#44535]
## Issue description
I'm trying to do some python development with tensorflow and tensorflow-tensorboard. I created the following shell.nix to create the right development environment.

```
with import (fetchTarball {
  url = https://github.com/nixos/nixpkgs-channels/archive/nixpkgs-unstable.tar.gz;
}) {};
mkShell {
  buildInputs =
    [(python3.withPackages (ps: with ps; [
      tensorflow
      tensorflow-tensorboard
    ]))];
}
```

Calling nix-shell doesn't work because tensorflow-tensorboard is currently broken.
Here's the log, made with `nix-shell --show-trace`:

```
error: while evaluating the attribute 'buildInputs' of the derivation 'nix-shell' at /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/stdenv/generic/make-derivation.nix:175:11:
while evaluating the attribute 'passAsFile' of the derivation 'python3-3.6.6-env' at /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/stdenv/generic/make-derivation.nix:175:11:
while evaluating 'requiredPythonModules' at /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/top-level/python-packages.nix:96:27, called from /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/development/interpreters/python/wrapper.nix:12:13:
while evaluating 'unique' at /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/lib/lists.nix:539:12, called from /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/top-level/python-packages.nix:98:6:
while evaluating 'requiredPythonModules' at /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/top-level/python-packages.nix:96:27, called from /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/top-level/python-packages.nix:114:33:
while evaluating 'unique' at /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/lib/lists.nix:539:12, called from /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/top-level/python-packages.nix:98:6:
while evaluating 'requiredPythonModules' at /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/top-level/python-packages.nix:96:27, called from /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/top-level/python-packages.nix:114:33:
while evaluating 'unique' at /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/lib/lists.nix:539:12, called from /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/top-level/python-packages.nix:98:6:
while evaluating 'hasPythonModule' at /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/top-level/python-packages.nix:93:21, called from undefined position:
while evaluating 'getOutput' at /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/lib/attrsets.nix:451:23, called from undefined position:
while evaluating anonymous function at /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/stdenv/generic/make-derivation.nix:129:17, called from undefined position:
while evaluating 'overridePythonAttrs' at /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/top-level/python-packages.nix:46:31, called from /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/top-level/python-packages.nix:1242:24:
while evaluating 'makeOverridablePythonPackage' at /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/top-level/python-packages.nix:40:37, called from /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/top-level/python-packages.nix:46:40:
while evaluating 'makeOverridable' at /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/lib/customisation.nix:67:24, called from /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/top-level/python-packages.nix:42:12:
while evaluating anonymous function at /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/development/interpreters/python/build-python-package.nix:27:1, called from /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/lib/customisation.nix:69:12:
while evaluating anonymous function at /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/development/interpreters/python/mk-python-derivation.nix:15:1, called from /nix/store/aj6yhbfyrc7l930rr4f7r50zk99b58b4-source/pkgs/development/interpreters/python/build-python-package.nix:44:4:
html5lib-0.9999999 not supported for interpreter python3.6m
```

### Steps to reproduce
See above.

## Technical details
I'm using NixOS 18.09 if that matters. I doubt it does because nix sandboxes are supposed to be reproducible on all systems with nix.
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the
results.

```
 - system: `"x86_64-linux"`
 - host os: `Linux 4.14.59, NixOS, 18.03.133029.d0c868ec17c (Impala)`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.0.4`
 - channels(root): `"nixos-18.03.133029.d0c868ec17c, nixos-unstable-18.09pre147850.2428f5dda13"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos/nixpkgs`
```

**Comment by dotlambda** [comment]
See https://github.com/NixOS/nixpkgs/issues/39588. However, you might want to try it with #42783 applied.

**Comment by dotlambda** [comment]
Fixed by #44599.
